### ISSUE TITLE
StandaloneRouting.js error when passing JSON

### ISSUE THREAD

**Issue by sctkndl** [OHIF/Viewers#1587]
I'm also trying to use the viewer with the attached JSON provided by Zaid Safadi who suggested I open this ticket. 

Using the current OHIF code from GIT, running it locally with node.js and yard as described in the setup guide, I was trying to use the  viewer with a local url json file
http://localhost:3000/viewer?url=http://localhost/webheart/dicom_json.json

I can see the json getting loaded, but it runs into some issues with StandaloneRouting.js (/platform/viewer/src/routes/)

I had to change some variables to get them to work on StandaloneRouting.js with this json
study.StudyInstanceUID -> study.studyInstanceUid
series.SeriesInstanceUID - > series.seriesInstanceUid
study.series -> study.seriesList

After getting past those errors, i got to this one:
There is nothing called metaData in instance in the json, so I'm not sure what to do there. It's failing on line 98:
const { url: imageId, metadata: naturalizedDicom } = instance;

So then the rest of the function won't work
                // Add instance to metadata provider.
                metadataProvider.addInstance(naturalizedDicom);
                // Add imageId specific mapping to this data as the URL isn't necessarliy WADO-URI.
                metadataProvider.addImageIdToUIDs(imageId, {
                  StudyInstanceUID,
                  SeriesInstanceUID,
                  SOPInstanceUID: naturalizedDicom.SOPInstanceUID,
                });

[dicom_json.zip](https://github.com/OHIF/Viewers/files/4411329/dicom_json.zip)

I assume Zaid's json is formatted correctly, but if it's not and there is some guide on how to do it properly, please let me know.

**Comment by dannyrb** [comment]
Are you using the new format called out in this announcement: https://github.com/OHIF/Viewers/issues/1500

**Comment by sctkndl** [comment]
No, and thank you for pointing this out. I have updated the json and now have success.
